Lib/fontTools/feaLib/parser.py

```diff
@@ -1062,6 +1128,15 @@ def expect_float_(self):
         raise FeatureLibError("Expected a floating-point number",
                               self.cur_token_location_)
 
+    def expect_decipoint_(self):
+        if self.next_token_type_ == Lexer.FLOAT:
+            return self.expect_float_() * 10
```

Comment:
I believe this is wrong, at least according to my reading of the [feature_file_syntax_spec#8.b](http://www.adobe.com/devnet/opentype/afdko/topic_feature_file_syntax.html#8.b), as well as the comments by @readroberts in https://github.com/adobe-type-tools/afdko/issues/112.

When there is an integer, it is interpreted as a "decipoint" (1/10 of a point); alternatively, this

> may also be specified more directly as decimal point values... For example, "8.0" and "80" will both result in the same value being stored in the font.

In your code, it's the opposite. When there's a float, you multiply it by 10, whereas when it's an integer you take it as is.
I believe you need to divide the integer by 10, and leave the float as is.